packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -845,10 +851,174 @@ export default class Meeting extends StatelessWebexPlugin {
+     * @type {Object}
+     * @property {string} captchaId captcha id
+     * @property {string} verificationImageURL Url of the captcha image
+     * @property {string} verificationAudioURL Url of the captcha audio file
+     * @property {string} refreshURL Url used for refreshing the captcha (don't use it directly, call refreshCaptcha() instead)
+     * @public
+     * @memberof Meeting
+     */
+    this.requiredCaptcha = null;
+
+    /**
+     * Indicates the reason for last failure to obtain meeting.meetingInfo. Undefined if meeting info was
+     * retrieved successfully
+     * @instance
+     * @type {MEETING_INFO_FAILURE_REASON}
+     * @public
+     * @memberof Meeting
+     */
+    this.meetingInfoFailureReason = undefined;
+
     this.setUpLocusInfoListeners();
     this.locusInfo.init(attrs.locus ? attrs.locus : {});
   }
 
+  /**
+   * Fetches meeting information.
+   * @param {Object} options
+   * @param {String} options.destination
+   * @param {String} options.type
+   * @private
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  async fetchMeetingInfo({
+    destination, type, password = null, captchaCode = null
+  }) {
+    if (captchaCode && !this.requiredCaptcha) {
+      return Promise.reject(new Error('captcha not required'));
+    }
+    if (password && (this.passwordStatus !== PASSWORD_STATUS.REQUIRED && this.passwordStatus !== PASSWORD_STATUS.UNKNOWN)) {
+      return Promise.reject(new Error('password not required'));
+    }
+
+    try {
+      const captchaInfo = captchaCode ? {code: captchaCode, id: this.requiredCaptcha.captchaId} : null;
+
+      const info = await this.attrs.meetingInfoProvider.fetchMeetingInfo(destination, type, password, captchaInfo);
+
+      this.parseMeetingInfo(info);
+      this.meetingInfo = info ? info.body : null;
+      this.meetingInfoFailureReason = MEETING_INFO_FAILURE_REASON.NONE;
+      this.requiredCaptcha = null;
+      if ((this.passwordStatus === PASSWORD_STATUS.REQUIRED) || (this.passwordStatus === PASSWORD_STATUS.VERIFIED)) {
+        this.passwordStatus = PASSWORD_STATUS.VERIFIED;
+      }
+      else {
+        this.passwordStatus = PASSWORD_STATUS.NOT_REQUIRED;
+      }
+
+      return Promise.resolve();
+    }
+    catch (err) {
+      if (err instanceof MeetingInfoV2PasswordError) {
+        LoggerProxy.logger.info(`Meeting:index#fetchMeetingInfo --> Info Unable to fetch meeting info for ${destination} - password required (code=${err?.body?.code}).`);
+
+        // when wbxappapi requires password it still populates partial meeting info in the response
+        if (err.meetingInfo) {
+          this.meetingInfo = err.meetingInfo;
+          this.meetingNumber = err.meetingInfo.meetingNumber;
+        }
+
+        this.passwordStatus = PASSWORD_STATUS.REQUIRED;
+        this.meetingInfoFailureReason = MEETING_INFO_FAILURE_REASON.WRONG_PASSWORD;
+        if (this.requiredCaptcha) {
+          // this is a workaround for captcha service bug, see WEBEX-224862
```

Comment:
if you pass a valid captcha and bad password it still markes it as failure 
?

Reply:
yes, it's a bug in wbxappapi: when you give them a good captcha with bad password they don't ask for captcha anymore, but they won't give you the meeting info even if you provide the good password in next attempt. The only way to make it work is to ask for another captcha - same workaround was done in UCF

Reply:
@marcin-bazyl do you have a follow up ticket to revisit this when it gets fixed in the API?

Reply:
ok, I've raised https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-293259 for this, but don't hold your breath for the wbxappapi fix.... ;-)